### PR TITLE
PD-325  Add more detailed steps to rejoin node in k8s

### PR DESCRIPTION
**Pull request by harsh-px**
Signed-off-by: Harsh Desai <harsh@portworx.com>

### CODE REVIEW

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md [hunk 1]
```diff
@@ -30,41 +28,51 @@ sudo rm -f /etc/systemd/system/portworx*
 grep -q '/opt/pwx/oci /opt/pwx/oci' /proc/self/mountinfo && sudo umount /opt/pwx/oci
 ```
 
-## Step 3: Cleanup Portworx metadata on the node
+## Clean up Portworx metadata on the node
+
+Portworx also stores local metadata on the node to fingerprint its identity. The method you used to decommission the node determines the steps you must follow to clean up local Portworx metadata.
+
+<!-- what are the circumstances under which pxctl is still on the node vs not? -->
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md]: `pxctl` would be absent from the node if user had manually deleted `/opt/pwx/bin` from the node. pxctl is a binary in this directly.

The doc aims to be idempotent. So I'm covering all cases/situations from which the user could end up here.

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md [hunk 2]
```diff
@@ -30,41 +28,51 @@ sudo rm -f /etc/systemd/system/portworx*
...
-```
+    ```text
+    wipefs -af /dev/sdf
+    ```
+
+## Restart the Portworx pod on your node by cleaning up labels
+
+Once you have completely decommissioned your node and wiped it of Portworx data, you're ready to re-add it to your cluster. Enter the following `kubectl label nodes` commands, replacing `<node>` with your node ID:
+
+<!-- where do users get the value for <node>? -->
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md]: you can rename `replacing `<node>` with your node ID` to 

`replacing `<node>` with the Kubernetes node name of this node`

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md [hunk 3]
```diff
@@ -2,33 +2,77 @@
...
+sudo systemctl disable portworx
+sudo rm -f /etc/systemd/system/portworx*
+grep -q '/opt/pwx/oci /opt/pwx/oci' /proc/self/mountinfo && sudo umount /opt/pwx/oci
+```
+
+## Clean up Portworx metadata on the node
+
+Portworx also stores local metadata on the node to fingerprint its identity. The method you used to decommission the node determines the steps you must follow to clean up local Portworx metadata.
+
+<!-- what are the circumstances under which pxctl is still on the node vs not? -->
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md]: @harsh-px

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md [hunk 4]
```diff
@@ -2,33 +2,77 @@
...
+    wipefs -af /dev/sdf
     ```
 
-In above command, _\<node\>_ is the node which is rejoining the cluster.
-After above steps, Portworx will start on this node and will join the cluster back as a new node.
+## Restart the Portworx pod on your node by cleaning up labels
+
+Once you have completely decommissioned your node and wiped it of Portworx data, you're ready to re-add it to your cluster. Enter the following `kubectl label nodes` commands, replacing `<node>` with your node ID:
+
+<!-- where do users get the value for <node>? -->
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md]: @harsh-px

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md (hunk 1 above)
Comment (nathan-px) [reply content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md]: Ok, so we can leave it as written?

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md (hunk 4 above)
Comment (harsh-px) [reply content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md]: Replied above

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md (hunk 3 above)
Comment (harsh-px) [reply content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/k8s-node-rejoin.md]: Replied above

**Comment by nathan-px** [comment]
LGTM. @harsh-px want me to merge this back to 2.2?

**Pull request merged**